django/db/backends/utils.py

```diff
@@ -209,3 +209,16 @@ def format_number(value, max_digits, decimal_places):
     if decimal_places is not None:
         return "%.*f" % (decimal_places, value)
     return "{:f}".format(value)
+
+
+def strip_trailing_quotes(table):
```

Comment:
I'd use: `def strip_quotes(table_name)` (considering the leading quote is removed also, not just the trailing one).